pkg/rpc/param_test.go

```diff
@@ -0,0 +1,42 @@
+package rpc
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParam_UnmarshalJSON(t *testing.T) {
+	msg := `["kek", 123, ["lul", 3]]`
```

Comment:
kek lul.. maaan! =) probably you can think of smth less fun?=)

Reply:
I will rename to "elem1" and "elem2". "kek" and "lul" are defaults which came to my mind first ;)